index.js

```diff
@@ -190,12 +185,13 @@ function loadConfig(program) {
         saveMessages.push(`Exclusions: ${processed.diff} been saved to config.`)
       }
 
-      if (program.excludeRegex) {
-        parsedConfig.exclude_regex = program.excludeRegex
-        // strip leading and trailing / if exists
-        if (parsedConfig.exclude_regex.substring(0, 1) === '/' && parsedConfig.exclude_regex.substring(parsedConfig.exclude_regex.length - 1) === '/') {
-          parsedConfig.exclude_regex = parsedConfig.exclude_regex.substring(1, parsedConfig.exclude_regex.length - 1)
-        }
+      if (program.excludeRegex || !isNaN(process.env.LOGDNA_EXCLUDE_REGEX)) {
+        let re = program.excludeRegex || process.env.LOGDNA_EXCLUDE_REGEX
+        // FIXME(darinspivey) - Error prone. Doesn't support modifiers or ending with
+        // a slash.  The user should be forced to provide open and closing slashes,
+        // otherwise it's too hard to know what's part of the pattern text.
+        re = re.replace(/^\//, '').replace(/\/$/, '')
+        parsedConfig.exclude_regex = re
```

Comment:
@darinspivey, why don't u combine these 2 lines - the current one with the one above?

Reply:
Can do

Reply:
fixed